.gitignore

```diff
@@ -1046,6 +1046,10 @@ platform-sdk/sdk/metricsDoc.tsv
 # Temporary Test Folders (Poorly Behaved Tests)
 **/swirlds-tmp
 swirlds-platform-core/data/**
+hedera-node/cli-clients/testFile
+hedera-node/hedera-mono-service/src/test/resources/balance*/**
+hedera-node/hedera-mono-service/temp/**
+hedera-node/hedera-mono-service/swirlds-sst-tmp/**
```

Comment:
Please open a ticket to fix the related tests which are creating temporary files in the production code structure instead of using temporary folders.

Reply:
Will do.